tools/cmake/cores/Cortex-M33FE-NS.cmake

```diff
@@ -7,11 +7,11 @@ if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
         "-mthumb"
         "-mfpu=fpv5-sp-d16"
         "-mfloat-abi=softfp"
-        "-march=armv8-m.main+dsp"
+        "-mcpu=cortex-m33+dsp"
```

Comment:
Is `+dsp` valid here? Maybe, but don't see it in the docs. They just say `cortex-m33` DSP is assumed, and you can specify `+nodsp`. (Note we're also not specifying `+fp`).

https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html

`-mcpu` works differently to `-march` in this regard - you're always enabling extensions in `-march`, but usually disabling them in `-mcpu`.

Reply:
I'll remove dsp from this one